src/test/scala/nl.knaw.dans.easy.bagstore/ItemIdSpec.scala

```diff
@@ -121,4 +161,13 @@ class ItemIdSpec extends TestSupportFixture {
       case Success(b) => b shouldBe bagId
     }
   }
+
+  private def expectValidationToFailWithMessage(nonsenseUuid: String, expectedMessage: String) = {
+    try {
+      validateUuid(nonsenseUuid)
+      fail("too short uuid should throw an illegal argument exception")
+    } catch {
+      case iae: IllegalArgumentException => assert(iae.getMessage.contains(expectedMessage))
```

Comment:
Something to study http://www.scalatest.org/user_guide

Reply:
Will do